examples/quickstart/jupyter-notebooks/api-tutorial.ipynb

```diff
@@ -0,0 +1,442 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f578b9b2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ingestion_taskId = json.loads(ingestiion_taskId_response.text)['taskId']\n",
+    "print(ingestion_taskId)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f17892d9-a8c1-43d6-890c-7d68cd792c72",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Get the status of your task\n",
+    "\n",
+    "The following cell shows you how to get the status of your ingestion.  You can see basic information about your query, such as when it started and whether or not it's finished.\n",
+    "\n",
+    "In addition to the status, you can retrieve a full report about it if you want using `GET /druid/indexer/v1/task/TASK_ID/reports`. But you won't need that information for this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fdbab6ae",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = f\"/druid/indexer/v1/task/{ingestion_taskId}/status\"\n",
+    "print(druid_host+endpoint)\n",
+    "http_method = \"GET\"\n",
+    "\n",
+    "payload = {}\n",
+    "headers = {}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "print(json.dumps(json.loads(response.text), indent=4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b55af57-9c79-4e45-a22c-438c1b94112e",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Query your data\n",
+    "\n",
+    "When you ingest data into Druid, Druid stores the data in a datasource, and this datasource is what you run queries against.\n",
+    "\n",
+    "### List your datasources\n",
+    "\n",
+    "You can get a list of datasources from the `/druid/coordinator/v1/datasources` endpoint. Since you're just getting started, there should only be a single datasource, the `wikipedia_api` table you created earlier when you ingested external data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "959e3c9b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "endpoint = \"/druid/coordinator/v1/datasources\"\n",
+    "print(druid_host+endpoint)\n",
+    "http_method = \"GET\"\n",
+    "\n",
+    "payload = {}\n",
+    "headers = {}\n",
+    "\n",
+    "response = requests.request(http_method, druid_host+endpoint, headers=headers, data=payload)\n",
+    "print(response.text)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "622f2158-75c9-4b12-bd8a-c92d30994c1f",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Query your data\n",
+    "\n",
+    "Now, you can query the data. Because this tutorial is running in Jupyter, make sure to limit the size of your query results using `LIMIT`. For example, the following cell selects all columns but limits the results to 3 rows for display purposes.\n"
```

Comment:
maybe mention that each row is a json object? Maybe mention that the tutorial uses select * but "in real life" only use the columns you need? Maybe link out to Druid SQL?